doc/release-notes-30529.md

```diff
@@ -0,0 +1,4 @@
+Configuration
+-------------
+
+Some corner cases handling negated list options `-norpcwhitelist`, `-norpcallowip`, `-norpcbind`, `-nobind`, `-nowhitebind`, `-noexternalip`, `-noonlynet`, `-noseednode`, `-nosignetchallenge`, `-nosignetseednode`, `-notest`, `-norpcwallet`, and `-no have been fixed. Now negating these options is the same as not specifying them.
```

Comment:
Typo:
```
`-norpcwallet`, and `-no have
```

Also not sure this should be included in this sentence assuming it is `-asmap` since it was not used as a list arg?